ui/lobby.lua

```diff
@@ -555,12 +555,22 @@ function G.UIDEF.create_UIBox_lobby_menu()
 									UIBox_button({
 										button = "view_code",
 										colour = G.C.PALE_GREEN,
-										minw = 3.15,
+										minw = 2.15,
 										minh = 1.35,
 										label = { localize("b_view_code") },
 										scale = text_scale * 1.2,
 										col = true,
 									}),
+									create_spacer(),
+									UIBox_button({
+										button = "copy_to_clipboard",
+										colour = G.C.PALE_GREEN,
+										minw = 1.35,
+										minh = 1.35,
+										label = { localize("b_copy_code"), localize("b_clipboard") },
```

Comment:
```suggestion
										label = localize("b_copy_code_clipboard"),
```

This should fix the use of the entry, I already tested this myself.